protocols/ELISA_immunoresearch_lab/ELISA_jackson_immunoresearch.ot2.py

```diff
@@ -0,0 +1,99 @@
+from opentrons import instruments, labware, robot
+
+# labware definitions
+tips_1000 = labware.load('tiprack-1000ul', '1')
+samples = labware.load('tube-rack-2ml', '2')
+output = labware.load('96-PCR-flat', '3')
+tips_50 = labware.load('tiprack-200ul', '4')
+trough = labware.load('trough-12row', '5')
+deepwell = labware.load('96-deep-well', '6')
+
+# instrument setup
+p50 = instruments.P50_Single(
+    tip_racks=[tips_50],
+    mount='left')
+
+p1000 = instruments.P1000_Single(
+    tip_racks=[tips_1000],
+    mount='right')
+
+# reagents and variables setup
+# setup for samples + initial two dilutions of 1:100 and 1:2500 in 2ml tuberack
+buff = trough.wells('A1')  # location of buffer in trough
+sam1 = samples.wells('A1')  # location of concentrated sample 1
+sam2 = samples.wells('B1')  # location of concetrated sample 2
+s1_dil1 = samples.wells('A2')  # location of 1:100 dilution of sample 1
+s2_dil1 = samples.wells('B2')  # location of 1:100 dilution of sample 2
+s1_dil2 = samples.wells('A3')  # location of 1:2500 dilution of sample 1
+s2_dil2 = samples.wells('B3')  # location of 1:2500 dilution of sample 2
+
+dil1_list = [s1_dil1, s2_dil1]  # locations of both 1:100 dilutions
+dil2_list = [s1_dil2, s2_dil2]  # locations of both 1:250 dilutions
+
+
+# ADD BUFFER TO 2ML RACK, DEEPWELL PLATE, AND OUTPUT PLATE
+# add buffer to dilution tubes in 2ml rack (wells A2, B2, A3, B3)
+p1000.pick_up_tip()
+for dil1, dil2 in zip(dil1_list, dil2_list):
+    p1000.transfer(495, buff, dil1, new_tip='never')
```

Comment:
This can be done without a for loop like this:

p1000.pick_up_tip()
p1000.transfer(vol, buff, dil1, new_tip='never')
p1000.transfer(vol, buff, dil2, new_tip='never')
p1000.drop_tip()